.github/workflows/taiko-client--hive-test.yml

```diff
@@ -2,7 +2,7 @@ name: "Taiko Client Hive Tests"
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+    types: [ opened, synchronize, reopened, ready_for_review ]
```

Comment:
As we discussed, we shall only run HIVE tests after merging a client PR?

Reply:
The hive test is triggered by `option.hive-test` tag.